packages/create/src/recipes/sveltekit/template/ts/src/lib/server/auth.__auth.ts

```diff
@@ -0,0 +1,6 @@
+import { createClient } from "edgedb";
```

Comment:
Maybe this file should be named something else? It only creates the edgedb client, so it's not specific to using auth, and is useful to include in the base svelte template. So maybe `lib/server/edgedb.ts`?

Reply:
yes, we used to create auth client here, will rename it